src/SOS/Strike/util.cpp

```diff
@@ -668,8 +684,12 @@ void DisplaySharedStatic(ULONG64 dwModuleDomainID, DacpMethodTableData* pMT, Dac
         DacpDomainLocalModuleData vDomainLocalModule;
         if (g_sos->GetDomainLocalModuleDataFromAppDomain(appdomainData.AppDomainPtr, (int)dwModuleDomainID, &vDomainLocalModule) != S_OK)
         {
-            DMLOut(" %s:NotInit ", DMLDomain(pArray[i]));
```

Comment:
FYI, this is one of the changes that fix the dumpobj statics bug.  The rest of the changes in util.cpp are the coreclr sync.